source/frontend/src/features/properties/list/PropertyListView.tsx

```diff
@@ -53,7 +52,19 @@ const PropertyListView: React.FC = () => {
   const [sort, setSort] = useState<TableSort<IProperty>>({});
 
   const fetchIdRef = useRef(0);
-
+  const history = useHistory();
+  const onShowPropertyInfo = useCallback(
+    (property: IProperty, inNewTab: boolean) => {
+      !inNewTab && history.push(`/mapview/sidebar/property/${property.id}?pid=${property.pid}`);
+      inNewTab &&
+        window.open(`/mapview/sidebar/property/${property.id}?pid=${property.pid}`, '_blank');
+    },
+    [history],
+  );
```

Comment:
This implementation is correct - however I would recommend a simpler approach where no callback is needed. Look at how **LeaseSearchResult** was implemented for an existing pattern on links for each table row. Example:

```tsx
// in colums.tsx
Cell: (props: CellProps<ILeaseSearchResult>) => (
      <Link to={`/lease/${props.row.original.id}`}>{props.row.original.lFileNo}</Link>
    ),
```

Notice that you already have the current property in context for each table row, no need to pass that to a callback function. Inlining a simple router **<Link ... />** is sufficient

Reply:
As for the link to open in a new tab:

```tsx
<Link to="route" target="_blank" rel="noopener noreferrer" />
```

Reply:
yes this is much better, changed.